sys/sys_amd64.const

```diff
@@ -56,7 +56,7 @@ DN_RENAME = 16
 EFD_CLOEXEC = 524288
 EFD_NONBLOCK = 2048
 EFD_SEMAPHORE = 1
-EPOLLET = 2147483648
+EPOLLET = 18446744071562067968
```

Comment:
How did this happen? What kernel version is this? EPOLLET is defined as 1<<31:
http://elixir.free-electrons.com/linux/latest/source/include/uapi/linux/eventpoll.h#L48
At least drop sys/sys_*.const changes from this patch.